config/dynamic.json

```
@@ -1,32 +1,27 @@
 {
-	"Dynamic": {},
+	"Dynamic": {
+		"roundstart_divergence_percent_lower": 0.8,
+		"roundstart_divergence_percent_upper": 1.4,
```

```suggestion
		"roundstart_divergence_percent_lower": 1,
		"roundstart_divergence_percent_upper": 1,
```
I don't want any divergence since its unpredictable